diff -Naur gnome-settings-daemon-3.20.1.orig/data/org.gnome.settings-daemon.plugins.xsettings.gschema.xml.in.in gnome-settings-daemon-3.20.1/data/org.gnome.settings-daemon.plugins.xsettings.gschema.xml.in.in
--- gnome-settings-daemon-3.20.1.orig/data/org.gnome.settings-daemon.plugins.xsettings.gschema.xml.in.in	2016-04-13 19:26:49.000000000 +0700
+++ gnome-settings-daemon-3.20.1/data/org.gnome.settings-daemon.plugins.xsettings.gschema.xml.in.in	2016-07-23 12:58:14.229857611 +0700
@@ -36,7 +36,7 @@
       <_description>A list of strings representing the GTK+ modules that will be loaded, usually in addition to conditional and forcibly disabled ones.</_description>
     </key>
     <key type="a{sv}" name="overrides">
-      <default>{}</default>
+      <default>{'Gtk/ButtonImages': <0>, 'Gtk/MenuImages': <1>}</default>
       <summary>A dictionary of XSETTINGS to override</summary>
       <description>This dictionary maps XSETTINGS names to overrides values. The values must be either strings, signed int32s or (in the case of colors), 4-tuples of uint16 (red, green, blue, alpha; 65535 is fully opaque).</description>
     </key>
